source/MaterialXView/Material.cpp

```diff
@@ -602,7 +602,7 @@ mx::ShaderPort* Material::findUniform(const std::string& path) const
             });
 
         // Check if the uniform exists in the shader program
-        if (port && !_uniformNames.count(port->getName()))
+        if (port && !_uniformNames.count(port->getVariable()))
```

Comment:
Since we are not actually using uniform names anymore, I think it would be good to rename the _uniformNames to _variableNames so it's clear what we're dealing with.

Reply:
Fixed!